Makefile

```diff
@@ -49,3 +49,7 @@ setup: all
 	@echo 'Generating API admin token'
 	hackillinois-utility-accountgen
 	hackillinois-utility-tokengen
+
+.PHONY: docs
+docs:
+	cd $(BASE_DIR)/documentation && $(MAKE) -C build && cd -
```

Comment:
Does this work? The purpose of the `-C` flag is so that we don't need to change directories to run make.

Reply:
I guess I'm not sure what $(MAKE) -C does. What does it do, and how does it know what target to hit / Makefile to run on?
```suggestion
	$(MAKE) -C build
```

Reply:
https://www.gnu.org/software/make/manual/html_node/Options-Summary.html

You'll probably want something along the lines of:
`$(MAKE) -C $(BASE_DIR)/documentation build`

But test it and make sure it works.

Reply:
Yes this works.